docs/standard/microservices-architecture/architect-microservice-container-applications/containerize-monolithic-applications.md

```diff
@@ -3,39 +3,39 @@ title: Containerizing monolithic applications
 description: .NET Microservices Architecture for Containerized .NET Applications | Containerizing monolithic applications
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 05/26/2017
+ms.date: 09/20/2018
 ---
 # Containerizing monolithic applications
 
-You might want to build a single, monolithically deployed web application or service and deploy it as a container. The application itself might not be internally monolithic, but structured as several libraries, components, or even layers (application layer, domain layer, data-access layer, etc.). Externally, however, it is a single container—a single process, a single web application, or a single service.
+You might want to build a single, monolithically deployed web application or service and deploy it as a container. The application itself might not be internally monolithic, but structured as several libraries, components, or even layers (application layer, domain layer, data-access layer, etc.). Externally, however, it is a single container�a single process, a single web application, or a single service.
 
-To manage this model, you deploy a single container to represent the application. To scale up, you just add more copies with a load balancer in front. The simplicity comes from managing a single deployment in a single container or VM.
+To manage this model, you deploy a single container to represent the application. To increase capacity, you scale out, i.e just add more copies with a load balancer in front. The simplicity comes from managing a single deployment in a single container or VM.
```

Comment:
we can't use i.e. - should be that is

Reply:
Ok. Is it the same with "**e.g.**"? is "**for example**" a suitable replacement?